src/chrome/content/rules/Radare.org.xml

```diff
@@ -0,0 +1,14 @@
+<!--
+	Connection refused:
+		- cloud.radare.org
+
+	Wrong server:
+		- cydia.radare.org
+-->
+
+<ruleset name="Radare.org">
+	<target host="radare.org" />
+	<target host="www.radare.org" />
+
```

Comment:
You can add `<securecookie host="^(www)\.radare\.org" name=".+" />` to exclude the problematic domains

Reply:
Added, thanks!